sealtk/gui/Player.cpp

```diff
@@ -94,7 +95,9 @@ Player::~Player()
 {
   // We need our context to be current so our OpenGL resources can be cleaned
   // up properly
-  makeCurrent();
```

Comment:
Why, thank you for silently eating my comment, github...

Strictly, calling `destroyResources` isn't needed; the dtors can handle it as long as the context is current. OTOH it's not really harmful, except that the comment is now slightly inaccurate. What about:
```c++
QTE_D();

// We need to clean up our textures while our context is current...
d->destroyResources();

/// ...and then ensure cleanup isn't called after we're halfway destroyed
disconnect(d->destroyResourcesConnection);
```

FWIW, my local patch looks like:
```diff
@@ -117,6 +119,11 @@ Player::Player(QWidget* parent)
 //-----------------------------------------------------------------------------
 Player::~Player()
 {
+  QTE_D();
+
+  // Ensure cleanup is not called after we've been destroyed
+  disconnect(d->cleanup);
+
   // We need our context to be current so our OpenGL resources can be cleaned
   // up properly
   makeCurrent();
```

Reply:
> Strictly, calling destroyResources isn't needed; the dtors can handle it as long as the context is current. OTOH it's not really harmful, except that the comment is now slightly inaccurate. What about:

Explicit > implicit. As you said, it's not harmful, and I'd rather make it clear what we're doing.

I can adjust the comment to make it more accurate.